zookeeper_dashboard/common.py

```diff
@@ -3,4 +3,13 @@
 from django.conf import settings
 
 def get_zookeeper_servers():
-    return os.getenv('ZOOKEEPER_SERVERS') or getattr(settings,'ZOOKEEPER_SERVERS')
+    return (
+        os.getenv('ZOOKEEPER_SERVERS')
+        or getattr(settings,'ZOOKEEPER_SERVERS')
+    )
+
+def get_zookeeper_servers_as_list():
+    return get_zookeeper_servers().split(',')
+
+def get_zookeeper_server(id):
+    return get_zookeeper_servers_as_list()[int(id)]
```

Comment:
Why have the function convert to an int for you vs making id -> index and specifying that an int must be passed in?  Convenience?